dnf/cli/commands/__init__.py

```diff
@@ -622,7 +622,7 @@ def run_on_repo(self):
     class RemoveOrReinstallSubCommand(SubCommand):
         """Implementation of the remove-or-reinstall sub-command."""
 
-        aliases = ('remove-or-reinstall',)
+        aliases = ('remove-or-reinstall', 'rm-ri',)
```

Comment:
Please do not make any abbreviation here. Better to know what you do here. Those command are so rare used that there is no need for it. I even didn't know that we have something like that.